engine/gamesys/src/gamesys/components/comp_particlefx.cpp

```diff
@@ -396,9 +400,33 @@ namespace dmGameSystem
                 pfx_world->m_VertexBufferData.SetSize(0);
                 pfx_world->m_RenderObjects.SetSize(0);
                 break;
-            case dmRender::RENDER_LIST_OPERATION_BATCH:
-                RenderBatch(pfx_world, params.m_Context, params.m_Buf, params.m_Begin, params.m_End);
-                break;
+            case dmRender::RENDER_LIST_OPERATION_BATCH: {
```

Comment:
I think you need to describe the problem and the solution in more detail, as I don't really know what we're trying to solve here.

Reply:
We've discussed some out of band. However, to give context here: I'm finding that max_particle_count can be very large in a game to accomodate worst case emitter and there is a copy in CPU as well as GPU. This patch is meant to keep the CPU side at max_particle_count and instead grow the GPU side to accomodate worst case rather than have them match and just drop on the floor.